### PR TITLE
update get start xpu

### PR DESCRIPTION
**Pull request by ZhaoqiongZ**
1. respect the comment from the community, downgrade the "Beta" to "Prototype" for the first xpu release with wheel 
2. add wheels installation of torchaudio & torchvision for nightly on Windows

### CODE REVIEW

**Comment by ZhaoqiongZ** [comment]
Hi @malfet , please help review this update!

**Review by atalman — APPROVED** [review]
lgtm

File: docs/source/notes/get_start_xpu.rst [hunk 1]
```diff
@@ -55,17 +55,17 @@ Platform Windows
...
 
-    pip3 install torch --index-url https://download.pytorch.org/whl/xpu
+    pip3 install torch --index-url https://download.pytorch.org/whl/test/xpu
 
 For nightly wheels
 
 .. code-block::
 
-    pip3 install --pre torch --index-url https://download.pytorch.org/whl/nightly/xpu
+    pip3 install --pre torch torchvision torchaudio --index-url https://download.pytorch.org/whl/nightly/xpu
```
Comment (kit1980) [inline docs/source/notes/get_start_xpu.rst:68]: Do you really want torchaudio here?

Comment (atalman) [reply docs/source/notes/get_start_xpu.rst:68]: I believe we do have both torchaudio and torchvision xpu builds: https://hud2.pytorch.org/hud/pytorch/audio/nightly/1?per_page=50&name_filter=xpu

**Review by malfet — APPROVED** [review]

**Pull request closed without merging**